src/Components/Form/Form.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+
+class Form extends Component {
+  state = {
+    errorMessage: '',
+  };
+
+  setErrorMessage = data =>
+    this.setState({
+      errorMessage: data,
+    });
+
+  submitForm = event => {
+    event.preventDefault();
+
+    const {
+      target: {
+        location: { value },
+      },
+    } = event;
+    const API_ENDPOINT = `${process.env.REACT_APP_WEATHER_API_URL}${value}${process.env.REACT_APP_WEATHER_API_KEY}`;
```

Comment:
you should declare this API_ENDPOINT in 
common/constants.js    file in the root directory of your project. And then just simply Import, and use it here.
This constant may be used many times across the project so by placing them in common you avoid repeating.

Reply:
I cant use it as a const cuz I neet to past in here value ${value} from form